src/con_utils.ts

```diff
@@ -123,6 +138,43 @@ function parseConnectDsnAndArgs({
     );
     serverSettings = server_settings;
   }
+
+  if (
+    !(
+      dsn ||
+      host ||
+      port ||
+      process.env.EDGEDB_HOST ||
+      process.env.EDGEDB_PORT
+    )
+  ) {
+    if (process.env.EDGEDB_INSTANCE) {
+      dsn = process.env.EDGEDB_INSTANCE;
+    } else {
+      const dir = process.cwd();
```

Comment:
And we probably need a shim for this one, deno has `Deno.cwd()`

Reply:
This one's ok, the `process` and `Buffer` globals are injected automatically

Reply:
Oh, nice